src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -41,7 +70,7 @@ public sealed class OptionalFieldAttribute : Attribute
     {
         public int VersionAdded
         {
-            get { return default(int); }
+            get { throw null; }
```

Comment:
I think we should switch throughout. It's easier to read because three's less noise to distract and easier to write because you don't have to repeat return and out parameter types. 

There is also a theoretical reduction in reference assembly size. I recall compiler folks saying that a single 'throw null' IL body could be reused/shared across the assembly.